keepassxc-browser/content/autocomplete.js

```diff
@@ -232,12 +229,10 @@ kpxcAutocomplete.updatePosition = function(inputField, elem) {
 
     const rect = inputField.getBoundingClientRect();
     div.style.minWidth = Pixels(inputField.offsetWidth);
-    const bodyRect = document.body.getBoundingClientRect();
-    const bodyStyle = getComputedStyle(document.body);
 
-    if (bodyStyle.position.toLowerCase() === 'relative') {
-        div.style.top = Pixels(rect.top - bodyRect.top + document.scrollingElement.scrollTop + inputField.offsetHeight);
-        div.style.left = Pixels(rect.left - bodyRect.left + document.scrollingElement.scrollLeft);
+    if (_bodyStyle.position.toLowerCase() === 'relative') {
```

Comment:
These are awkward global variables, would be better to encapsulate then somewhere, perhaps in kpxcUI.